overlord/configstate/config/transaction_test.go

```diff
@@ -62,7 +64,9 @@ func (op setGetOp) args() map[string]interface{} {
 		}
 		kv := strings.SplitN(pair, "=", 2)
 		var v interface{}
-		err := json.Unmarshal([]byte(kv[1]), &v)
+		dec := json.NewDecoder(strings.NewReader(kv[1]))
+		dec.UseNumber()
+		err := dec.Decode(&v)
```

Comment:
Not related to the PR, but this could be written in a single line: `if err := dec.Decode(&v); err != nil {`